arches_lingo/src/arches_lingo/components/scheme/report/SchemeSection.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+import { useGettext } from "vue3-gettext";
+import Button from "primevue/button";
+
+const { $gettext } = useGettext();
+const props = defineProps<{
+    titleText: string;
+}>();
+
+const buttonText = $gettext(`Add ${props.titleText}`);
```

Comment:
gettext interpolation needs to be updated

https://jshmrtn.github.io/vue3-gettext/functions.html#interpolation

Also IMO you should inline this instead of declaring it.